3-enrich/stream-enrich/kinesis/src/main/scala/com.snowplowanalytics.snowplow.enrich.stream/KinesisEnrich.scala

```diff
@@ -184,14 +183,14 @@ object KinesisEnrich extends Enrich {
    */
   private def lookupDynamoDBResolver(
     provider: AWSCredentialsProvider,
-    region: String,
+    endpoint: String,
     table: String,
     key: String
   ): Validation[String, String] = {
     val dynamoDBClient = AmazonDynamoDBClientBuilder
       .standard()
       .withCredentials(provider)
-      .withEndpointConfiguration(new EndpointConfiguration(getDynamodbEndpoint(region), region))
+      .withEndpointConfiguration(new EndpointConfiguration(endpoint, endpoint))
```

Comment:
does this actually work? If so, great, but I would have thought that the second parameter of this EndpointConfiguration constructor had to be a region value, not an endpoint value.

Reply:
Same as above, I ran the debugger with a breakpoint at this line and it doesn't get hit. Seems to me that the changes to this file aren't needed